identity/backend/src/main/resources/application.yaml

```diff
@@ -35,3 +35,9 @@ server:
       force: true
     context-path: ${IDENTITY_BASE_PATH:}
   max-http-request-header-size: ${SERVER_REQUEST_HEADER_MAX_SIZE:8KB}
+camunda:
+  presets:
+    identity:
+      user: admin
+      password: admin
+
```

Comment:
With an array this would be
```yaml
camunda:
  identity:
    init:
      users:
        - username: admin
          password: demo
        - username: demo
          password: demo
```

Reply:
Ok, I see. I was thinking that we are going to create one admin user (the role part is in a separate task) and then will use it to access identity and setup other users and so on. 
But if we want this config to make the identity ready to use without the need for ui, I'll go for the change.

Reply:
It's "yes, but" 😄 You are right that we just need to create a user. In the past we have used the same functionality to create multiple users for integration/e2e (mostly e2e since int he integration tests you can create them in the code itself) tests and then the same code also developed to work as environment initialization. While, of course, all of this is out of the scope for this PR, we should still make the current approach extensible. IMO the easiest way of doing it is just to allow creating multiple users instead of only one